docker-compose.yml

```
@@ -1,6 +1,23 @@
 version: "3.9"
    
 services:
+
+  mysql:
+    image: mysql:5.7
+    restart: on-failure
```

We've wanted to change this `restart: on-failure` to `restart: unless-stopped`. I'd make this change here for this new project. https://docs.docker.com/config/containers/start-containers-automatically/

```suggestion
    restart: unless-stopped
```